MAUI/PDF-Viewer/Coordinate-Conversion.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: Working with PDF coordinate systems in MAUI PDF Viewer control | Syncfusion
+description: Learn here all about coordinate conversion in Syncfusion .NET MAUI PDF Viewer (SfPdfViewer) control for precise positioning and enhanced interaction. 
+platform: MAUI
+control: SfPdfViewer
+documentation: ug
+---
+
+# Working with PDF coordinate systems in .NET MAUI PDF Viewer (SfPdfViewer)
+
+PDF Viewer allows users to obtain the PDF page coordinates relative to the PDF Viewer’s client coordinates and vice versa. It also allows you to obtain the scroll point relative to the PDF page coordinates and bring the given region into view.
+
+## Client rectangle
+
+The MAUI PDF Viewer the dimensions of the client area(viewport) are known as client rectangle, which is the bounds of the control. The client rectangle in the MAUI PDF Viewer is represented by the red rectangle in the following figure.
+
+![Coordinate conversion in .NET MAUI PDF Viewer](Images\Coordinate-Conversion\Client-Rectangle.jpg)
```

Comment:
<img width="365" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/65169077/0308c6a0-f63a-4238-b0e4-23306ba06782">

The text should explain the image, not the topic. Please change in all the places

Reply:
addressed